### ISSUE TITLE
OAuth 2.0 support

### ISSUE THREAD

**Issue by papo-o** [maximebf/python-withings#22]
Nokia is sunsetting OAuth 1.0 support on November 30th, someone can update this library  to OAuth 2.0?

**Comment by robmarkcole** [comment]
Yes this is required, docs [here](http://developer.withings.com/oauth2/#tag/OAuth-2.0)

**Comment by mrueg** [comment]
There is an active fork here: https://github.com/orcasgit/python-nokia which provides oauth2
